Annotato/Annotato/WebSocket/WebSocketManager.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import AnnotatoSharedLibrary
+
+class WebSocketManager {
+    var socket: URLSessionWebSocketTask?
+
+    func setSocket(to session: URLSessionWebSocketTask) {
+        socket = session
+        self.listen()
+        socket?.resume()
+
+        print("Set socket")
```

Comment:
Are these print statements supposed to be here? If yes should we use the logger instead?

Reply:
No.  Will remove it later during integration.